src/Microsoft.Azure.WebJobs.Host/Listeners/FunctionListener.cs

```diff
@@ -138,7 +138,11 @@ public async Task StopAsync(CancellationToken cancellationToken)
                 {
                     if (_started)
                     {
+                        _logger?.LogInformation($"Stopping the listener for function '{_descriptor.ShortName}'");
```

Comment:
Could you write out the type of the Listener here as well? That could be helpful for debugging.